### PR TITLE
Vritant

### PR DESCRIPTION
**Pull request by vritant24**
Spiced up UI

### CODE REVIEW

File: public/map.js [hunk 1]
```diff
@@ -86,3 +86,24 @@ var selectLocation = function(feature) {
...
 };
+
+var toggleText = 0;
+
+$('#map-submit').click(function() {
+  if(toggleText == 0) {
+    $('#map').fadeOut('slow', function() {
+      $('#graph-body').fadeIn('slow');
+      $('#form-data').fadeIn('slow');
+      $('#map-submit').text('Select Another Location');
```
Comment (drewatk) [inline public/map.js]: You can use `this` here

File: public/map.js [hunk 2]
```diff
@@ -86,3 +86,24 @@ var selectLocation = function(feature) {
...
+    });
+    toggleText = 1;
+  } else {
+    $('#graph-body').fadeOut('slow', function() {
+      $('#map').fadeIn('slow');
+      $('#form-data').fadeOut('slow');
+      $('#map-submit').text('Confirm Selection');
```
Comment (drewatk) [inline public/map.js]: `this`

File: views/index.ejs [hunk 3]
```diff
@@ -4,41 +4,34 @@
...
+                </div>
             </div>
             <div class="col-md-9">
               <div class="row">
-                <div id="graph" class="col-md-12">
-                    <!-- This is where Google Charts will go -->
-                    <h3>Graph</h3>
+                <div id="map" class="col-md-12">
+                  <!-- This is where Google Maps will go -->
+                  <h3>Map</h3>
```
Comment (drewatk) [inline views/index.ejs]: Do we need this h3?

File: public/map.js (hunk 1 above)
Comment (drewatk) [inline public/map.js]: Nope, disregard

File: public/map.js (hunk 2 above)
Comment (drewatk) [inline public/map.js]: Nope, disregard

File: views/index.ejs (hunk 3 above)
Comment (vritant24) [inline views/index.ejs]: Fixed, removed

File: public/map.js (hunk 2 above)
Comment (vritant24) [inline public/map.js]: No

File: public/map.js (hunk 1 above)
Comment (vritant24) [inline public/map.js]: Disregarded

**Pull request merged**